Resources/Private/Templates/OaiPmh/Main.xml

```diff
@@ -122,7 +124,9 @@
                         </f:else>
                     </f:if>
                 </f:for>
-                <resumptionToken cursor="{resumptionToken.cursor}" completeListSize="{resumptionToken.completeListSize}" expirationDate="<f:format.date format="Y-m-d\TH:i:s\Z">{resumptionToken.expired}</f:format.date>">{resumptionToken.token}</resumptionToken>
+                <f:if condition="{resumptionToken.token}">
```

Comment:
The `resumptionToken` is missing now on the last "page".

In the linked "FlowControl" it's written:

> The repository responds with an incomplete list of 75 records. The repository marks this list as the final incomplete list by including in the response an empty resumptionToken element with two attributes: a completeListSize of 175, and a cursor of 100.

Reply:
Sorry for having missed that. Do you think the patch is right? As far as I understood, the `<resumptionToken>` tag should still be omitted if there is only one page, and there shouldn't be an `expirationDate` parameter on the last page. So I've extracted this to a separate section in the Fluid template.